src/ui/view/share/data/dropDown/dropDown.tsx

```diff
@@ -0,0 +1,137 @@
+import type { FC } from 'react'
+import { useMemo, useCallback, useState } from 'react'
+import { Collapsible } from '@/ui/component/collapsible/collapsible'
+import { SearchBar } from '@/ui/component/searchbar/searchbar'
+import { isSubstringOf } from '@/util/util'
+import { DynamicCheckbox } from '@/ui/view/dataverse/component/dynamicCheckbox/dynamicCheckbox'
+import { NoResultFound } from '@/ui/view/dataverse/component/noResultFound/noResultFound'
+import { Tag } from '../../../../component/tag/tag'
+import './dropDown.scss'
+
+type DropDownOption = {
+  value: string
+  disabled?: boolean
+  selected?: boolean
+}
+
+type SelectionItemType = 'checkbox'
+
+type DropDownProps = {
+  value: DropDownOption[]
```

Comment:
In this context, `value` must be an array of strings. `selected` option state can be managed internally :)